caffe2/serialize/inline_container.cc

```diff
@@ -612,6 +612,24 @@ size_t ostream_write_func(
   return ret;
 }
 
+// This seeking function pretends to behave like m_pWrite, but will actually
+// seek n bytes from the current offset
```

Comment:
I'm not sure about this comment tbh. This is just doing the regular seek function?

From the write function above, the only thing I would have a comment about is the crc32 handling?